src/state/entities/environments/selectors.ts

```diff
@@ -8,3 +9,8 @@ export const getEnvironmentsForDropdown = (state: IState) => {
     const environments = asyncEnvironments.data || [] as IEnvironment[];
     return environments.map((environment) => environment.name);
 };
+
+export const getScriptForDropdown = () => {
+    const scriptExecutionStatus = Object.values(ExecutionActionStatus);
+    return scriptExecutionStatus.map((status) => status);
+};
```

Comment:
move to where ExecutionActionStatus is defined.
Also rename function name to more correct name